Source/Core/Core/PowerPC/PPCAnalyst.cpp

```diff
@@ -191,18 +192,19 @@ static void AnalyzeFunction2(Common::Symbol* func)
   func->flags = flags;
 }
 
-static bool CanSwapAdjacentOps(const CodeOp& a, const CodeOp& b)
+static bool CanSwapAdjacentOps(const CodeOp& a, const CodeOp& b, bool is_debugging_enabled)
```

Comment:
This could be made into a member function instead.

Reply:
Sure, I guess.